test/csuite/incr_backup/main.c

```diff
@@ -0,0 +1,869 @@
+    testutil_check(conn->close(conn, NULL));
+}
+
+int
+main(int argc, char *argv[])
+{
+    ACTIVE_FILES active;
+    TABLE_INFO tinfo;
+    WT_CONNECTION *conn;
+    WT_RAND_STATE rnd;
+    WT_SESSION *session;
+    uint32_t file_max, iter, max_value_size, next_checkpoint, rough_size, slot;
+    int ch, ncheckpoints, status;
+    const char *working_dir;
+    char conf[1024], home[1024], backup_check[1024], backup_dir[1024], command[4096];
+
+    ncheckpoints = 0;
+    (void)testutil_set_progname(argv);
+    custom_die = die; /* Set our own abort handler */
+    WT_CLEAR(tinfo);
+    active_files_init(&active);
+
+    working_dir = "WT_TEST.incr_backup";
+
+    while ((ch = __wt_getopt(progname, argc, argv, "h:S:v:")) != EOF)
+        switch (ch) {
+        case 'h':
+            working_dir = __wt_optarg;
+            break;
+        case 'S':
+            seed = (uint64_t)atoll(__wt_optarg);
+            break;
+        case 'v':
+            verbose_level = atoi(__wt_optarg);
+            break;
+        default:
+            usage();
+        }
+    argc -= __wt_optind;
+    if (argc != 0)
+        usage();
+
+    if (seed == 0) {
+        __wt_random_init_seed(NULL, &rnd);
+        seed = rnd.v;
+    } else
+        rnd.v = seed;
+
+    testutil_work_dir_from_path(home, sizeof(home), working_dir);
+    testutil_check(__wt_snprintf(backup_dir, sizeof(backup_dir), "%s.BACKUP", home));
+    testutil_check(__wt_snprintf(backup_check, sizeof(backup_check), "%s.CHECK", home));
+    printf("Seed: %" PRIu64 "\n", seed);
+
+    testutil_check(
+      __wt_snprintf(command, sizeof(command), "rm -rf %s %s; mkdir %s", home, backup_dir, home));
+    if ((status = system(command)) < 0)
+        testutil_die(status, "system: %s", command);
+
+    /*
+     * We create an overall max_value_size.  From that, we'll set a random max_value_size per table.
+     * In addition, individual values put into each table vary randomly in size, up to the
+     * max_value_size of the table.
+     * This tends to make sure that 1) each table has a "personality" of size ranges within it
+     * 2) there are some runs that tend to have a lot more data than other runs.  If we made every
+     * insert choose a uniform random size between 1 and MAX_VALUE_SIZE, once we did a bunch
+     * of inserts, each run would look very much the same with respect to value size.
+     */
+    max_value_size = __wt_random(&rnd) % MAX_VALUE_SIZE;
+
+    /* Compute a random value of file_max. */
+    rough_size = __wt_random(&rnd) % 3;
+    if (rough_size == 0)
+        file_max = 100 + __wt_random(&rnd) % 100; /* small log files, min 100K */
+    else if (rough_size == 1)
+        file_max = 200 + __wt_random(&rnd) % 1000; /* 200K to ~1M */
+    else
+        file_max = 1000 + __wt_random(&rnd) % 20000; /* 1M to ~20M */
+    testutil_check(__wt_snprintf(
+      conf, sizeof(conf), "create,log=(enabled=true,file_max=%" PRIu32 "K)", file_max));
```

Comment:
One suggestion is that at some verbose level you might also add `verbose=(backup)` to the config. I just did this exact thing to debug the CREATE issue.